package.json

```diff
@@ -10,6 +10,7 @@
       "import": "./build/three.module.js",
       "require": "./build/three.cjs"
     },
+    "./addons": "./examples/jsm/addons.js",
```

Comment:
What do you think about a `build/addons.module.js` or `build/addons.cjs`, like what you have for three above?

Reply:
aaah you're right, but some libraries exported in `addons.js` don't work in a commonjs environment. Could you test it out?

Reply:
Do you mean stuff like DRACO or examples/js/libs? Not sure what you want me to test or if there's a script we can come up with.

Reply:
some libraries in `examples/jsm/libs` access the window, this breaks node, how is that handled in a commonjs environment?

For example, `TTFLoader` breaks in node because its library is accessing the window.

Reply:
I see, that's a bit tricky. Usually you can check with `if (typeof window !== 'undefined') window.foo = bar`, but it's unclear how to handle that for multiple libs or if rollup can do that for us.

Might be something for an issue/another PRs. I know three-stdlib solved this, but they import those libs from NPM instead.

Reply:
Our `examples/jsm/libs` folder contains on ES Module libs, so it can't be easily converted to a commonjs environment.

Also, the `examples/jsm` folder is an es-module itself, and unless we add a build step, it can't be exposed as commonjs.

We have never provided a commonjs output for the examples js files, just `esm` and `iife`. So I'd avoid staring to do that especially because now more and more people are using `esm` whitin node.